samples/subsys/usb/hid-cdc/src/main.c

```diff
@@ -620,6 +626,13 @@ void main(void)
 
 	usb_hid_register_device(hid1_dev, hid_kbd_report_desc,
 				sizeof(hid_kbd_report_desc), &ops);
+
+	ret = usb_enable(status_cb);
```

Comment:
So to use this callback we shall call usb_enable(cb) always?

Reply:
Yes. If callback is needed then USB_DEVICE_AUTO_ENABLE=n and call usb_enable(cb) in the app.